sky/packages/sky/lib/src/widgets/basic.dart

```diff
@@ -899,8 +899,15 @@ class WidgetToRenderBoxAdapter extends LeafRenderObjectWrapper {
 // EVENT HANDLING
 
 class IgnorePointer extends OneChildRenderObjectWrapper {
-  IgnorePointer({ Key key, Widget child })
+  IgnorePointer({ Key key, Widget child, this.ignoring })
```

Comment:
We probably want to give a default `ignoring` value here too:

`IgnorePointer({ Key key, Widget child, this.ignoring: true })`